ci-operator/step-registry/optional-operators/ci/run-bundle/optional-operators-ci-run-bundle-commands.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Deploying an operator in the bundle format using operator-sdk run bundle command"
+
+cd /tmp
+operator-sdk run bundle "${OO_BUNDLE_IMG}" \
+                        --index-image "${OO_INDEX_IMG}" \
+                        --namespace "${OO_INSTALL_NAMESPACE}" \
+                        --install-mode "${OO_TARGET_NAMESPACES}"
```

Comment:
Can this fail? If it does, do we need to collect some artifacts for debugging?

Reply:
When `run bundle` is pulling the bundle image and extracting its contents, it would write those to ROOT, which would cause a permission denied error. So just changing the working directory to `tmp` instead.

Reply:
Added collection of artifacts after `run bundle` is executed.